legend-engine-server/src/main/java/org/finos/legend/engine/server/Server.java

```diff
@@ -118,6 +120,8 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(
         PureProtocolObjectMapperFactory.withPureProtocolExtensions(bootstrap.getObjectMapper());
         VaultFactory.withVaultConfigurationExtensions(bootstrap.getObjectMapper());
         ObjectMapperFactory.withStandardConfigurations(bootstrap.getObjectMapper());
+
+        bootstrap.getObjectMapper().registerSubtypes(new NamedType(LegendDefaultDatabaseAuthenticationFlowProviderConfiguration.class, "legendDefault"));
```

Comment:
I think subtype registration can be done in a similar way like protocol/vault extensions

Reply:
I'm ok with unifying the configuration wiring. However, to keep this MR small, I'd rather that we address the config refactoring in a separate MR.